packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -560,6 +570,60 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	updateFocusAfterVoiceControl = () => {
+		const
+			{direction} = this.props,
+			viewportBounds = this.uiRef.containerRef.getBoundingClientRect(),
+			spotItemBounds = document.activeElement.getBoundingClientRect(),
+			nodes = this.uiRef.containerRef.querySelectorAll('[data-index]'),
+			isVertical = (direction === 'vertical' || direction === 'both'),
+			first = isVertical ? 'top' : 'left',
+			last = isVertical ? 'bottom' : 'right';
+
+		if (spotItemBounds[last] < viewportBounds[first] || spotItemBounds[first] > viewportBounds[last]) {
+			for (let i = 0; i < nodes.length; i++) {
+				const nodeBounds = nodes[i].getBoundingClientRect();
+				if (nodeBounds[first] > viewportBounds[first] && nodeBounds[last] < viewportBounds[last]) {
+					Spotlight.focus(nodes[i]);
+					break;
+				}
+			}
+		}
+	}
+
+	onVoice = (e) => {
+		const {scroll} = e.detail;
+		this.isVoiceControl = true;
```

Comment:
What if a browser passes a wrong e.detail value? We do not do anything in this function, but we tries to focus in scrollStopOnScroll () -> updateFocusAfterVoiceControl()

Reply:
Add *default:* condition in switch statement.